KDLoadingView/KDLoadingExtension.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  KDLoadingExtension.swift
+//  KDLoadingView
+//
+//  Created by Rodrigo Soldi Lopes on 02/03/17.
+//  Copyright © 2017 Kaique Damato. All rights reserved.
+//
+
+extension KDLoadingView {
+    
+    public class func animate(lineWidth: CGFloat = 2.0, size: CGFloat = 25, duration: CGFloat = 3.0, firstColor: UIColor? = #colorLiteral(red: 0, green: 0, blue: 0, alpha: 1), secondColor: UIColor? = nil, thirdColor: UIColor? = nil) {
+        guard let window = UIApplication.shared.keyWindow,
+            let topView = window.rootViewController?.view else {
+                return
+        }
+        
+        // Blur View
+        let blurView = KDLoadingBlurView(effect: UIBlurEffect(style: .light))
+        blurView.frame = topView.frame
+        
+        let frame = CGRect(x: 0, y: 0, width: size, height: size)
+        let loadingView = KDLoadingView(frame: frame, lineWidth: lineWidth, firstColor: firstColor, secondColor: secondColor, thirdColor: thirdColor, duration: duration)
+        loadingView.center = blurView.center
+        
+        blurView.addSubview(loadingView)
+        loadingView.startAnimating()
+        blurView.loadingView = loadingView
+        
+        addSubviewWithTransitionAnimation(fromView: topView, toView: blurView)
+    }
+    
+    public class func stop() {
+        if let window = UIApplication.shared.keyWindow {
```

Comment:
Maybe it's better to replace if let for guard let and put some log explaining the problem in else statement